pkgs/development/libraries/pipewire/default.nix

```diff
@@ -62,14 +62,13 @@
 , openssl
 , rocSupport ? true
 , roc-toolkit
-, x11Support ? true
+, x11Support ? true, ffado
 , libcanberra
 , xorg
 , mysofaSupport ? true
 , libmysofa
 , tinycompress
-, ffadoSupport ? stdenv.buildPlatform.canExecute stdenv.hostPlatform
-, ffado
+, ffadoSupport ? x11Support && stdenv.buildPlatform.canExecute stdenv.hostPlatform
```

Comment:
On what is this based? I couldn't find something in the meson config.

Reply:
This is simply disabling ffado support in Pipewire if a build without x11 support is required.
The `ffado` program requires at least `qtbase` and `gtk3` to build:
```
  ffado = libsForQt5.callPackage ../os-specific/linux/ffado {
    inherit (linuxPackages) kernel;
  };
```